ckanext/versioning/tests/test_functional.py

```diff
@@ -86,7 +86,6 @@ def test_package_show_renders_alert_info(self):
         url = toolkit.url_for(
             'versioning.show',
             package_id=self.dataset['id'],
-
             revision_ref=tag['revision_ref'])
```

Comment:
Should we be running a similar test but with `tag['name']` in addition to this one?